src/components/book.js

```diff
@@ -1,18 +1,18 @@
 import React from 'react';
```

Comment:
The project looks awesome! But README is missing the project title and description. You may use [this](https://github.com/microverseinc/readme-template) README template from Microverse.

Reply:
Thanks @SajjadAhmad14 ! I’ll definitely will do.